Aliases/Server - Heroism & Shame/dt/animal.alias

```diff
@@ -0,0 +1,101 @@
+embed -thumb <image> -color <color> -footer "Usage - !dt animal -name \"name\" [adv] [-rr #]"
+<drac2>
+# sets primary variables
+me, pargs, returnstr, trainComplete, trainCounter = character(), argparse(&ARGS&), "", False, 0
+
+# sets the booleans
+withAdv = True if pargs.get("adv") else False
+hasAnimal = True if exists("DTAnimalTrain") else False
+choseAnimal = True if pargs.get("name") else False
+
+# sets roll string
+rollStr = "1d20+" if not withAdv else "2d20kh1+"
```

Comment:
`rollStr = ['1d20+', '2d20kh1+', '2d20kl1+'][pargs.adv()]`

This automates `adv/dis` parsing, as `pargs.adv()` returns 0 for normal, 1 for adv and 2 or -1 for dis (-1 being the last element in an array)